impl/src/main/java/org/ehcache/internal/store/heap/OnHeapStore.java

```diff
@@ -174,7 +176,8 @@ public OnHeapStore(final Configuration<K, V> config, TimeSource timeSource, Copi
   }
   
   private OnHeapValueHolder<V> internalGet(final K key, final boolean updateAccess) throws CacheAccessException {
-    return map.computeIfPresent(key, new BiFunction<K, OnHeapValueHolder<V>, OnHeapValueHolder<V>>() {
+    accessObserver.begin();
```

Comment:
I'm ignorant of lots of the details here... but it seems weird that we're calling the `accessObserver` here, but not calling the `tierEfficiencyObserver`.